fastn/src/tauri.rs

```diff
@@ -0,0 +1,33 @@
+pub fn run(fastn_serve_port: u16) {
+    // fastn main has setup tokio so we make tauri use that
+    tauri::async_runtime::set(tokio::runtime::Handle::current());
+
+    tauri::Builder::default()
+        .setup(move |app| {
+            use tauri::Manager;
+
+            if cfg!(debug_assertions) {
+                app.handle().plugin(
+                    tauri_plugin_log::Builder::default()
+                        .level(log::LevelFilter::Info)
+                        .build(),
+                )?;
+            }
+
+            let webview = app
+                .get_webview_window("main")
+                .expect("main label is created by tauri");
+
+            let url = format!("http://localhost:{fastn_serve_port}")
```

Comment:
Over my years of experience I have come to prefer 127.0.0.1. Either that or localhost should be the same. But sometimes it is not the same (/etc/hosts entry, people are funny). And every time they are not the same I wish I should have used 127.0.0.1. We can not assume every single place fastn is installed has not done anything funny in their /etc/hosts, and we know for sure not matter whats in their /etc/hosts, 127.0.0.1 will always be what you want, so 127.0.0.1 is the preferred always over localhost.

Reply:
done in https://github.com/fastn-stack/fastn/pull/2165/commits/6f85671f7f4c435db23dac161776cca6d1de423f